lpc82x/src/lib.rs

```diff
@@ -0,0 +1 @@
+compile_error!("The `lpc82x` crate is deprecated. Please use `lpc82x-hal` or `lpc82x-pac` instead.");
```

Comment:
Nice macro!

Reply:
Yeah :-)
I wish it was possible to generate a custom warning the same way, but I don't think a warning macro exists.